src/main/java/conversion/BaseServlet.java

```diff
@@ -180,7 +203,100 @@ abstract void convert(final Individual individual, final Map<String, String[]> p
             final String inputDirectory, final String outputDirectory,
             final String fileNameWithoutExt, final String ext, final String contextURL);
 
-    private String getFileName(final Part part) {
+    /**
+     * Gets array of bytes from a url.
+     *
+     * @param strUrl
+     * @return the bytes downloaded from the url, null if no bytes downloaded.
+     * @throws IOException
+     */
+    private static byte[] getFileFromUrl(final String strUrl) throws IOException {
+
+        final int bufferSize = 1024; // Good buffer size?
```

Comment:
Looks fine to me!
Again, probs best to leave comments like this as a comment on the PR

Reply:
Will remember to comment on PR's from now on instead of in code :)